compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/CMakeLists.txt

```diff
@@ -40,6 +40,9 @@ iree_cc_library(
     "IREEGPUInterfaces.cpp"
     "IREEGPUOps.cpp"
     "TilingInterfaceImpl.cpp"
+  DEFINES
+    $<$<PLATFORM_ID:Windows>:MLIR_CAPI_ENABLE_WINDOWS_DLL_DECLSPEC>
+    $<$<PLATFORM_ID:Windows>:MLIR_CAPI_BUILDING_LIBRARY>
```

Comment:
This works (at least locally - gonna run win CI 🤞) but I doubt it's the "correct" thing to do.